tests/foreman/cli/test_content_credentials.py

```diff
@@ -0,0 +1,879 @@
+# -*- encoding: utf-8 -*-
+"""Test class for GPG Key CLI.
+The gpg sub-command was deprecated in favour of content-credentials in
+Satellite 6.8
+
+:Requirement: content_credentials
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: content_credentials
```

Comment:
There's no such component, sorry.
That should be "ContentCredentials".
And while you're at it, please add it to `testimony.yaml`, under `CaseComponent` key. Values there are supposed to be sorted alphabetically (testimony doesn't care, but it's easier for people when order of items is maintained).

Reply:
OK, and also :Requirement: content_credentials
 needs to be changed I think.

Reply:
@mirekdlugosz , OK, changes pushed

Thank you

Reply:
Requirement field is easier, as it's not transformed or validated in any way. Any string will suffice. If you pick requirement that doesn't exist in Polarion, Jenkins will create it for you during automation run.
Best to go to Polarion and pick one of requirements we already have. If you can't find any close enough, creating new one is OK.